conf/messages

```diff
@@ -1141,6 +1141,21 @@ itemCommodityCode.defaultItem = Choose commodity code
 
 itemQuantity.title = How much {0} are you moving?
 itemQuantity.heading = How much {0} are you moving?
+itemQuantity.label = How much {0} are you moving?
+itemQuantity.hint.liquid = Enter the total quantity of liquids in {0}.
+
+itemQuantity.title.T200 = How many thousands of cigarettes are you moving?
+itemQuantity.heading.T200 = Number of cigarettes you are moving?
+itemQuantity.paragraph.T200 = The excise duty is worked out per 1000 cigarettes. You can work out what to enter by dividing the total quantity of cigarettes you are moving by 1000. This does not mean the number of packets.
+itemQuantity.label.T200 = How many thousands of cigarettes are you moving?
+itemQuantity.hint.T200 = For example, enter 5 for 5000 cigarettes, or enter 1234.567 for 1,234,567 cigarettes.
+
+itemQuantity.title.T300 = How many thousands of cigars or cigarillos are you moving?
```

Comment:
As above r.e. shouldn't this match the h1 not the label?

Reply:
It could do although I thought the title should have the same wording as the label **How many thousands of cigars or cigarillos are you moving** otherwise the title is going to be **Number of cigars or cigarillos you are moving**

I felt the nature of this ticket was to be more explicit that for T300's we are actually asking how many cigars or cigarillos expressed as thousands and that the title should match this explicitness.

Reply:
but it doesn't match the prototype so i'll change it

Reply:
https://github.com/hmrc/emcs-tfe-create-movement-frontend/pull/394/commits/6bd06284620c60275b052a360b8886cadfa61491